### PR TITLE
Add New FAQ Q&A for AMA Migration Topic

### PR DESCRIPTION
**Pull request by jtracey93**
## Overview/Summary

Add New FAQ Q&A for AMA Migration Topic

## This PR fixes/adds/changes/removes

1. Closes [AB#25746](https://dev.azure.com/CSUSolEng/2a9d5a5a-8998-4ad0-81c8-ef3045e4da97/_workitems/edit/25746)
2. Amend `Update Portal` Workflow to trigger only when required

### Breaking Changes

None, just docs

## Testing Evidence

Linting will suffice as just docs.

## As part of this Pull Request I have

- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Enterprise-Scale/pulls)
- [x] Associated it with relevant [issues](https://github.com/Azure/Enterprise-Scale/issues), for tracking and closure.
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Enterprise-Scale/tree/main)
- [x] Performed testing and provided evidence.
- [x] Ensured [contribution guidance](https://github.com/Azure/Enterprise-Scale/wiki/ALZ-Contribution-Guide) is followed.
- [x] Updated relevant and associated documentation.
- [x] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located: `/docs/wiki/whats-new.md`)

### CODE REVIEW

File: docs/wiki/FAQ.md [hunk 1]
```diff
@@ -152,3 +153,19 @@ The Management Group Names/IDs created via the ALZ Portal Accelerator Deployment
...
   - `<Resource Prefix (Root ID)>-decommissioned`
   - `<Resource Prefix (Root ID)>-sandbox`
+
+## Why hasn't Azure landing zones migrated to the Azure Monitor Agent yet?
+
+Great question! Don't worry we are aware of this required migration and change to Azure landing zones with the Log Analytics Agent (Microsoft Monitoring Agent - MMA) being retired in August 2024 as detailed here: [Migrate to Azure Monitor Agent from Log Analytics agent](https://learn.microsoft.com/azure/azure-monitor/agents/azure-monitor-agent-migration).
+
+We are working hard internally with the Azure Monitor Product Group (PG) to ensure everything that Azure landing zones requires and gets from the Log Analytics Agent (Microsoft Monitoring Agent - MMA) approach today is covered and has a path for migration to the Azure Monitor Agent (AMA) approach. This has been underway for sometime and continues to progress.
+
+The AMA agent bring a number of number concepts, resources and changes to existing integrations with other services, such as Microsoft Defender for Cloud, that all require validation by each of the associated PGs as well as the Azure landing zone team, prior to migrating to AMA from MMA.
```
Comment (paulgrimley) [inline docs/wiki/FAQ.md:163]: Can't add a suggestion for some reason but change 'number' to 'new'

File: docs/wiki/FAQ.md [hunk 2]
```diff
@@ -152,3 +153,19 @@ The Management Group Names/IDs created via the ALZ Portal Accelerator Deployment
...
+
+We will, when ready, provide Azure landing zones specific migration guidance that supports the existing and to be created PG documentation. We will also make the relevant changes to each of the implementation options (Portal, Bicep, Terraform) to support the migration, especially for greenfield scenarios.
```
Comment (paulgrimley) [inline docs/wiki/FAQ.md:165]: remove supports 'the' existing

**Review by paulgrimley — CHANGES_REQUESTED** [review]
some minor tweaks but good to get this one closed off in interim

File: docs/wiki/FAQ.md (hunk 1 above)
Comment (jtracey93) [reply docs/wiki/FAQ.md:163]: done

File: docs/wiki/FAQ.md (hunk 2 above)
Comment (jtracey93) [reply docs/wiki/FAQ.md:165]: done

File: docs/wiki/FAQ.md [hunk 3]
```diff
@@ -152,3 +153,19 @@ The Management Group Names/IDs created via the ALZ Portal Accelerator Deployment
...
   - `<Resource Prefix (Root ID)>-decommissioned`
   - `<Resource Prefix (Root ID)>-sandbox`
+
+## Why hasn't Azure landing zones migrated to the Azure Monitor Agent yet?
+
+Great question! Don't worry we are aware of this required migration and change to Azure landing zones with the Log Analytics Agent (Microsoft Monitoring Agent - MMA) being retired in August 2024 as detailed here: [Migrate to Azure Monitor Agent from Log Analytics agent](https://learn.microsoft.com/azure/azure-monitor/agents/azure-monitor-agent-migration).
+
+We are working hard internally with the Azure Monitor Product Group (PG) to ensure everything that Azure landing zones requires and gets from the Log Analytics Agent (Microsoft Monitoring Agent - MMA) approach today is covered and has a path for migration to the Azure Monitor Agent (AMA) approach. This has been underway for sometime and continues to progress.
+
+The AMA agent bring a number of new concepts, resources and changes to existing integrations with other services, such as Microsoft Defender for Cloud, that all require validation by each of the associated PGs as well as the Azure landing zone team, prior to migrating to AMA from MMA.
```
Comment (Springstone) [inline docs/wiki/FAQ.md:163]: brings not bring

Comment (jtracey93) [reply docs/wiki/FAQ.md]: done

**Review by matt-FFFFFF — APPROVED** [review]
LGTM

**Pull request merged**